core/deployment/src/main/java/io/quarkus/deployment/builditem/substrate/ReflectiveIgnoreWarningBuildItem.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.deployment.builditem.substrate;
+
+import org.jboss.jandex.DotName;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+/**
+ * Used to ignore reflection warning deliberately
+ */
+public final class ReflectiveIgnoreWarningBuildItem extends MultiBuildItem {
```

Comment:
Maybe we should call it `ReflectiveHierarchyIgnoreWarningBuildItem` and also mention in the javadoc that it's used in the ReflectiveHierarchyBuildStep?

Reply:
will do !

Reply:
:+1:

Reply:
done